pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/controls/PlayerBottomToolbar.kt

```diff
@@ -58,10 +58,12 @@ import ch.srgssr.pillarbox.demo.shared.R as sharedR
 @Composable
 fun PlayerBottomToolbar(
     modifier: Modifier = Modifier,
-    shuffleEnabled: Boolean,
-    onShuffleClick: (() -> Unit)?,
+    isShuffleEnabled: Boolean,
+    isShuffleOn: Boolean,
+    onShuffleClick: () -> Unit,
+    isRepeatEnabled: Boolean,
     repeatMode: @Player.RepeatMode Int,
-    onRepeatClick: (() -> Unit)?,
+    onRepeatClick: () -> Unit,
```

Comment:
We this implementation the button are always present even if the player can't do them. It is fine but it's change the view behavior. For exemple Media3 `CastPlayer`